config/initializers/content_security_policy.rb

```diff
@@ -0,0 +1,10 @@
+# This file is overwritten during deployment
```

Comment:
I was wondering that, but I was nervous about setting it to true here and then having it be enabled in production if the file wasn't overwritten during deployment. Happy to take it out if you want.